File: Programming Basics/Conditional Statements Advanced - Exercises/01.Cinema.cs
﻿using System;

class Program
{
    static void Main(string[] args)
    {
        string type = Console.ReadLine();
        int rows = int.Parse(Console.ReadLine());
        int columns = int.Parse(Console.ReadLine());

        switch (type)
        {
            case "Premiere":
                Console.WriteLine($"{(rows * columns) * 12:f2} leva");
                break;
            case "Normal":
                Console.WriteLine($"{(rows * columns) * 7.50:f2} leva");
                break;
            case "Discount":
                Console.WriteLine($"{(rows * columns) * 5:f2} leva");
                break;
        }
    }
}

File: Programming Basics - Python/Conditional Statements Advanced - Exercises/01.Cinema.py
projection_type = input()
rows = int(input())
columns = int(input())

capacity = rows * columns

income = 0

if projection_type == 'Premiere':
    income = capacity * 12
elif projection_type == 'Normal':
    income = capacity * 7.50
elif projection_type == 'Discount':
    income = capacity * 5

print(f'{income:.2f}')
print('leva')